### PR TITLE
Add --encrypted flag to mysql creation

### LINKED ISSUE #2449: add `--encrypted` option for mysql resources
Looks like this is supported for Redis and Prostgres by not mysql. Just for kicks I passed `--encrypted` and got an `encrypted=true` in the confirmation message but when I check the AWS console, the instance does not have encryption enabled.

### PR DESCRIPTION
**Pull request by brandonblock**
Fixes  #2449

### CODE REVIEW

File: provider/aws/templates/resource/mysql.tmpl [hunk 1]
```diff
@@ -109,6 +118,33 @@
           "StorageType": "gp2",
           "VPCSecurityGroups": [ { "Ref": "SecurityGroup" } ]
         }
+      },
+            "EncryptionKey": {
```
Comment (ddollar) [inline provider/aws/templates/resource/mysql.tmpl]: Please fix up the indentation here

File: cmd/convox/resources.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ var resourceTypes = []ResourceType{
 	},
 	{
 		"mysql",
-		"[--allocated-storage=10] [--database=db-name] [--instance-type=db.t2.micro] [--multi-az] [--password=example] [--private] [--username=example] [--version=5.7.16]",
+		"[--allocated-storage=10] [--database=db-name] [--instance-type=db.t2.micro] [--multi-az] [--password=example] [--private] [--encrypted] [--username=example] [--version=5.7.16]",
```
Comment (ddollar) [inline cmd/convox/resources.go]: Please put the `--encrypted` flag in alphabetical order

**Comment by brandonblock** [comment]
@ddollar Comments addressed

**Comment by ddollar** [comment]
Thanks!

**Pull request closed without merging**